backend-peer/src/lib/error-handler/index.ts

```diff
@@ -32,11 +32,20 @@ interface IInternalError {
 export class HttpResponse extends Error {
   public code: number
   public message: string
-
-  constructor({ code = 500, message = 'Internal server error' }) {
+  public details: string
+  public stack: string
+
+  constructor({
+    code = 500,
+    message = 'Internal server error',
+    details = '',
+    stack = '',
```

Comment:
but I might be missing something

Reply:
So Error and it's subclasses already have the `stack` attribute (it's a getter, because generating a trace is a heavy process). Because of that, we shouldn't be shadowing it by defining our own `stack` attribute, but rather only reading the existing attribute when needed (i.e. when logging errors)

Reply:
thanks, I've removed the stack shadow